ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/recovery/transactions/TransactionCache.java

```diff
@@ -311,6 +316,10 @@ private static void remove (Uid theUid)
         }
 	}
     }
+
+    public static enum ReplayPhaseReturnStatus {
+        NONE, ASSUMED_COMPLETED, CACHE_REMOVED
```

Comment:
agree, I'm not satisfied with this way of enum definition. I wanted to change it when the behavior is working. Thank you for pointing it I will change the naming with usage of only two values.

With that I was considering to use only boolean (and I think you mention it) but it seems to me using enum is better in way of showing  what is the returning type. A {{boolean}} says nothing what is the meaning of the return value. But if you think it's much verbose I can change it to {{true/false}}.